charts/spire/templates/spire-controller-manager-config.yaml

```
@@ -0,0 +1,32 @@
+{{- if .Values.controllerManager.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "spire.fullname" . }}-controller-manager-config
+  namespace: {{ .Release.Namespace }}
+data:
+  spire-controller-manager-config.yaml: |-
```

```suggestion
  spire-controller-manager-config.yaml: |
```

I think without the `-` like in other configmaps to prevent whitespaces from being trimmed